exercises/ch04/exercise_b.js

```diff
@@ -1,4 +1,4 @@
 // Refactor to remove all arguments by partially applying the functions.
 
 // filterQs :: [String] -> [String]
-const filterQs = xs => filter(x => x.match(/q/i), xs);
```

Comment:
This change seems appropriate. Again, renaming the library function `match` to `test`, as well as this change, would be less confusing for js devs.